src/components/_modal/pages/bookmarkOrder/EMaterialFilter/EMaterialAnalyzer.js

```diff
@@ -40,10 +40,8 @@ const useAnalyzeMaterial = (material) => {
   );
   const physicalCopy = checkPhysicalCopy([access?.[0]])?.[0];
   const digitalCopy = checkDigitalCopy([access?.[0]])?.[0];
-  const onlineMaterialWithoutLoginOrLoginAtUrl = Boolean(
-    access?.length > 0 && !digitalCopy && !physicalCopy
-  );
-  return onlineMaterialWithoutLoginOrLoginAtUrl;
+
+  return !digitalCopy && !physicalCopy;
```

Comment:
OBS. den der tjekker kun den første access (om den er `!digitalCopy && !physicalCopy`, hhv. `DigitalArticleService` og `InterLibraryLoan`, så hvis der f.eks. er en `OnlineAccess`, så vil den her returnere false. Hvis det er hensigten, så er det fint, men bare vær opmærksom på, at hook'ens kommentar siger "returns whether the material is avaible online or not"  --> Problemet er så, at hvis der kun er `SpecialAccess`, så vil den returnere true, selvom det nok(!) ikke er tilfældet.

Reply:
Hvis det er et digitalt materiale som ikke er tilgængeligt, ønsker vi stadig true her, da vi ønsker at filtrere alle digetale materialer ud af denne bestil.